packages/portal/src/components/generic/CiteAttribution.vue

```diff
@@ -119,13 +119,26 @@
       extended: {
         type: Boolean,
         default: false
+      },
+      /**
+       * If `true`, expanded using keydown event
+       */
+      keyboardNav: {
+        type: Boolean,
+        default: false
       }
     },
 
     computed: {
       linkText() {
         return [this.name, this.creator, this.provider].filter(Boolean).join(', ');
       }
+    },
+
+    mounted() {
+      if (this.extended && this.keyboardNav) {
+        this.$el.getElementsByTagName('a')[0].focus();
```

Comment:
I think for "normal" pages this is in fact always going to be present, the only reason it might not be is when in preview mode.
No harm in having a null check though, that's been added.